tests/subyt/test_generator.py

```
@@ -9,18 +9,20 @@
 
 
 class AssertingSink(Sink):
-    def __init__(self):
+    def __init__(self, parts: list = None):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)